src/Config/ObjectModel/ParameterMetadata.cs

```diff
@@ -0,0 +1,13 @@
+namespace Azure.DataApiBuilder.Config.ObjectModel
+{
+    /// <summary>
+    /// Represents metadata for a parameter, including its name, description, requirement status, and default value.
+    /// </summary>
+    public class ParameterMetadata
+    {
+        public required string Name { get; set; }
```

Comment:
Dont we need to provide the Json property to these member properties? like `name` and `description`.